service/history/shard/context_impl.go

```diff
@@ -162,6 +162,7 @@ const (
 	logWarnTransferLevelDiff = 3000000 // 3 million
 	logWarnTimerLevelDiff    = time.Duration(30 * time.Minute)
 	historySizeLogThreshold  = 10 * 1024 * 1024
+	minContextTimeout        = 1 * time.Second
```

Comment:
2 seconds?

Reply:
sure, no strong opinion on this. Updated to 2s.